src/components/LandingPage/ArticleCard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import parseDate from '../../utils/neededFiles';
+
+export class ArticleCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
why is this here @kdaglas

Reply:
@kdaglas 👆  Currently we don't need that state.